### ISSUE TITLE
[Bug][Jupyter Widget] - Unable to Render the Map (Without Data Too) : Jupyter Notebook

### ISSUE THREAD

**Issue by pyJain108** [keplergl/kepler.gl#1587]
I am trying to plot a map using KeplerGl but there are no maps being rendered. I used the same data a month back, and it worked fine. I tried to load a map using GeoDataFrame and also without any data, but unable to see any output. 

**To Reproduce**
1. load Kepler
2. Add data & config
3. Export map

**Expected behavior**
A map should be displayed

**Screenshots**

![image](https://user-images.githubusercontent.com/75158219/131215537-540ef070-f760-4108-8f37-ed0f6a266434.png)

**Environment (please complete the following information):**
 - Python version: Python 3
 - Jupyter Notebook version: 6.4.3
 - OS: Windows

**Comment by pyJain108** [comment]
Surprisingly the map was rendered today. Please find the screenshot below : 
(**Note:** I didn't make any system changes, nor ran any additional command, I just restarted my system this morning and it worked)

Since this issue has been resolved, I will close the issue. 

![image](https://user-images.githubusercontent.com/75158219/131246270-fd58a252-57e2-4855-9c5b-7e071cf10176.png)

**Comment by pyJain108** [comment]
Apologies, have to reopen the issue - The map is not being rendered again :(  
The behaviour seems to be quite inconsistent. 
Jupyter Lab is also having the same issue, please find the screenshot below: 
![image](https://user-images.githubusercontent.com/75158219/131314652-4e70e108-d77a-43c1-9b0e-c6ce9c2f8d40.png)

**Comment by lixun910** [comment]
@pyJain108 Is there any error message in the browser console?

**Comment by pyJain108** [comment]
Hello @lixun910, 

The map is being rendered fine today  (Please find attached the screenshot) : 
(I will paste the error message of the console if the map rendering fails) 
![image](https://user-images.githubusercontent.com/75158219/131478943-828fc398-a659-4756-8418-7fe32c478d3f.png)

**Comment by sl-victormazzeo** [comment]
Hi, I have the same issue and I don't find any way to resolve it. I have the menu but not the map.
It seems that no token is passed but I can't find an option to give my personal token with the jupyterlab extension.

`keplergl.min.js:404 GET https://api.mapbox.com/styles/v1/mapbox/satellite-v9?pluginName=Keplergl&access_token=null 401 (Unauthorized)`

Any idea? Thx!

N.B. I am using JupyterLab 3.1.12 and keplergl 0.3.1

**Comment by lixun910** [comment]
@sl-victormazzeo Yes, it is because the keplergl v0.3.1 was built without a proper mapbox access token, see this pr: https://github.com/keplergl/kepler.gl/pull/1619. You can use the old keplergl==0.3.0 version, or wait for @heshan0131  to tag and release the keplergl v0.3.2. Thanks!

**Comment by heshan0131** [comment]
Just pushed `v0.3.2-jupyter` tag, @lixun910 could you check the new release once the workflow completes?

**Comment by rahatfcc** [comment]
@heshan0131 , Hi. By when can we expect v0.3.2 release in pypi.org ? I was making some developments on my existing product. Should I go with v0.3.0 for now?

**Comment by lixun910** [comment]
Thanks, @heshan0131 ! 👍 I did a quick: 
1. the keplergl v0.3.2 package has been published on PyPi https://pypi.org/project/keplergl/
2. it works fine on Jupyter Notebook, JupyterLab and Google Colab. 
3. keplergl-jupyter v0.3.2 npm package has also been published on npm:  https://www.npmjs.com/package/keplergl-jupyter, even though the github action shows an error:
https://github.com/keplergl/kepler.gl/runs/3617500852?check_suite_focus=true
```
npm ERR! code E404
npm ERR! 404 Not Found - PUT https://registry.npmjs.org/keplergl-jupyter - Not found
npm ERR! 404 
npm ERR! 404  'keplergl-jupyter@0.3.2' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-09-16T04_34_25_434Z-debug.log
Error: Process completed with exit code 1.
```

I am still waiting for conda-forge to finish the auto-build of v0.3.2 keplergl. I will check it once it's done.

**Comment by sl-victormazzeo** [comment]
Everything is working fine for me now. Thanks!

**Comment by lixun910** [comment]
@heshan0131 Tested! keplergl v0.3.2 is on conda-forge now: https://anaconda.org/conda-forge/keplergl

**Comment by doronbartov** [comment]
Kepler not able to render the plots on jupyter notebook.
My ipywidgets is 7.6.5 
kepler version is 0.3.2
saving to html works, but is less convenient cause the map opens always on california 

Would appreciate any help

**Comment by jingyan-li** [comment]
Hi I got the same issue  on jupyter notebook: 
![image](https://user-images.githubusercontent.com/37017983/169412998-94b20578-65ce-4df4-ac56-906d06cbc194.png)
And no map view shows up.

Kepergl version: 0.3.2
Ipywidgets : 7.7.0

Here is the error message in the browser console:
![image](https://user-images.githubusercontent.com/37017983/169413147-68336201-67c5-47bc-abfa-dac2ab5b9f44.png)

Currently, `map.save_to_html()` works for me. But I cannot adjust the configs and save configs in the HTML file for future rendering.

Thank you for any help!

**Comment by DimitrisMilonopoulos** [comment]
Any update on this issue?

**Comment by joaopcoelho** [comment]
I'm also having the same issue using kepler==0.3.2.

Uninstalling and installing 0.3.0 instead worked. I'd still be interested in understanding the reason behind this.

**Comment by vsincholle** [comment]
I had the same issue and downgrading from 0.3.2 to 0.3.0 solved issue for me too

**Comment by carolinsmemory** [comment]
It seems like this issue might resolve itself within a couple of days, but for now, neither version (0.3.2 or 0.3.0) shows a base map for me. They are either black or white, no boarders, water, roads etc.

**Comment by hazmimarzukian** [comment]
> I had the same issue and downgrading from 0.3.2 to 0.3.0 solved issue for me too

i did try to downgrade but the same problem still persists.
